❤️ , thanks, mister, for the fast PR. Do you mind adding the missing test?
> There is no test for the true → true path?  

In the pipeline, we see the error below.
Once done, I'll gladly merge your PR.

```
/home/runner/work/webdriverio/webdriverio/node_modules/.pnpm/esbuild@0.27.3/node_modules/esbuild/lib/main.js:1467
  let error = new Error(text);
              ^

Error: Build failed with 1 error:
error: [webdriver] Failed to generate d.ts files: TypeScript compilation failed with code 2
Error: src/bidi/remoteTypes.ts(503,17): error TS2552: Cannot find name 'BrowsingContextScreencast'. Did you mean 'BrowsingContextStopScreencast'?

--- a/packages/wdio-browserstack-service/tests/testHub/utils.test.ts
+++ b/packages/wdio-browserstack-service/tests/testHub/utils.test.ts
@@ -36,6 +36,11 @@ describe('getProductMap', () => {
         const productMap = utils.getProductMap({ ...config, accessibility: undefined } as any)
         expect(productMap.accessibility).toBe(false)
     })
+
+    it('should preserve explicit accessibility: true', () => {
+        const productMap = utils.getProductMap({ ...config, accessibility: true } as any)
+        expect(productMap.accessibility).toBe(true)
+    })
 })
 
 describe('shouldProcessEventForTesthub', () => {

